jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2502,6 +2502,59 @@ YUI.add('juju-env-api', function(Y) {
     },
 
     /**
+      Return the authorized SSH keys for the specified user.
+
+      @method listKeys
+      @param {String} user The user for whom the keys must be listed.
+      @param {Boolean} requestFullKeys Whether to pass full keys to the
+        callback, or just the SSH key fingerprints.
```

Comment:
While the effect of this argument is that the calback gets the fingerprint or the full key, the root cause is that it's requesting it differently from the api.